CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ https://github.com/nwnxee/unified/compare/build8193.6...HEAD
 - Util: SetInstructionLimit()
 - Area: CreateGenericTrigger()
 - Object: GetPositionIsInTrigger()
+- Administration: GetServerName()
```

Comment:
It's under the wrong header now. Move it to under [Unreleased] :D

Reply:
Not sure I understand - its showing under the [Unreleased] section, under the 
##### New NWScript Functions

Do you want it added to the ### Added  section?

Reply:
Oh, you probably have to pull the updated changelog.md from upstream

Reply:
Just rebased now - will do the same on the Faction PR too
Will wait until this one is merged - so I don't have to re-base twice etc